### PR TITLE
More Dutch translations

### LINKED ISSUE #3: Translation in Dutch
We need some people to translate this project.
See [how to translate](https://github.com/pampanic/pam_panic/wiki/How-to-translate)

### PR DESCRIPTION
**Pull request by karottenbaum**
### Proposed changes
- Add translations for the additional strings from #87. 

Fixes #3.

### CODE REVIEW

File: po/nl.po [hunk 1]
```diff
@@ -98,279 +98,305 @@ msgstr "Wachtwoord was niet gelijk. Probeer het nog eens.\n"
...
 
 #: src/pam_panic_config/pam_panic_config.prebuild.sh:79
 msgid "At your next login you need to"
-msgstr ""
+msgstr "Bij de volgende login moet je"
 
 #: src/pam_panic_config/pam_panic_config.prebuild.sh:80
 msgid "type your pam_panic authentification password or"
-msgstr ""
+msgstr "het pam_panic authentificatie-wachtwoord intypen of"
```
Comment (Bandie) [inline po/nl.po]: Do you really say "intypen"? :D

File: po/nl.po [hunk 2]
```diff
@@ -98,279 +98,305 @@ msgstr "Wachtwoord was niet gelijk. Probeer het nog eens.\n"
 
 #: src/pam_panic_config/pam_panic_config.prebuild.sh:18
 msgid "pam_panic"
-msgstr ""
+msgstr "pam_panic"
 
 #: src/pam_panic_config/pam_panic_config.prebuild.sh:21
 msgid "ERROR: Bash script was not built correctly."
-msgstr ""
+msgstr "FOUT: het Bash-script is niet correct gebouwd."
```
[the remark this answers is not in the document]
Comment (karottenbaum) [reply po/nl.po]: Fixed!

File: po/nl.po (hunk 1 above)
Comment (karottenbaum) [reply po/nl.po]: Yes, one can say that :D

File: po/nl.po [hunk 3]
```diff
@@ -98,279 +98,305 @@ msgstr "Wachtwoord was niet gelijk. Probeer het nog eens.\n"
...
 #: src/pam_panic_config/pam_panic_config.prebuild.sh:32
 msgid "Please insert the device you want to use as Panic device and press OK."
 msgstr ""
+"Steek nu het toestel in dat je als paniek-toestel gebruiken wil en klik dan "
+"OK."
 
 #: src/pam_panic_config/pam_panic_config.prebuild.sh:33
 msgid "Removable media: Authentication device"
-msgstr ""
+msgstr "Verwijderbare media: authenticatie-toestel"
```
Comment (Bandie) [inline po/nl.po]: Oh, I found something else .. can it be like this? Or is this wrong?
```suggestion
msgstr "Verwijderbare media: Authenticatie-toestel"
```

File: po/nl.po [hunk 4]
```diff
@@ -98,279 +98,305 @@ msgstr "Wachtwoord was niet gelijk. Probeer het nog eens.\n"
...
 
 #: src/pam_panic_config/pam_panic_config.prebuild.sh:34
 msgid "Removable media: Panic device"
-msgstr ""
+msgstr "Verwijderbare media: paniek-toestel"
```
Comment (Bandie) [inline po/nl.po]: (like above)
```suggestion
msgstr "Verwijderbare media: Paniek-toestel"
```

File: po/nl.po (hunk 1 above)
Comment (Bandie) [reply po/nl.po]: Alright~ :)

File: po/nl.po (hunk 2 above)
[the remark this answers is not in the document]
Comment (Bandie) [reply po/nl.po]: Thanks! :)

**Pull request merged**